lib/Runtime/Library/JavascriptFunction.cpp

```diff
@@ -3246,7 +3247,7 @@ void __cdecl _alloca_probe_16()
         if (proxy)
         {
             ParseableFunctionInfo * func = proxy->EnsureDeserialized();
-            return LiteralString::NewCopySz(func->GetDisplayName(), scriptContext);
+            return JavascriptString::NewCopySz(func->GetDisplayName(), scriptContext);
```

Comment:
@boingoing It seems like PFI::GetDisplayName sometimes returns a non-recycler-allocated buffer, so we can't use NewWithSz here. Is that accurate?

Reply:
True. I will do that instead.